### DISCUSSION TITLE
Save "Workspace"

### DISCUSSION THREAD

**Discussion by jfbaro** [nextapps-de/winbox#157] [Ideas]
Hello there.

Congrats on the great work you have done. Thanks!!

Can we have a feature where each (logged) user could arrange the many windows in a way that make more sense to them, and "SAVE" that "workspace, so that next time that they load the host application (the one that have the windows inside) the windows would be shown the way they looked last time.

Thanks!!

**Comment by ts-thomas** [comment]
This could be done by custom code. You can save windows positions and width/height. The content from a window very depends on your application, how this content will applied. But it is definitely possible to restore also every content. The logic when applying content to the window (during creation) should run again when restore a state.
